datasync/chaindatafetcher/config.go

```diff
@@ -18,6 +18,11 @@ package chaindatafetcher
 
 import "github.com/klaytn/klaytn/datasync/chaindatafetcher/kas"
 
+const (
+	ModeKAS = iota
```

Comment:
How about using enum type instead of literal int?

Reply:
Applied. 0c43a58d9c845d11b61d8d1f9bb035b22bc5f2eb